### PR TITLE
Always fetch the latest revision of the alfonse_specs from Bitbucket

### PR DESCRIPTION
**Pull request by monofon**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ download_khronos:
 
 # TODO: Get rid of revision number. How to get the newest revision from bitbucket?
```
Comment (jayschwa) [inline Makefile]: The TODO note is no longer relevant after this change and should be removed.

Comment (monofon) [inline Makefile]: Right. Thanks.

**Comment by chsc** [comment]
Thank's for the patch. I'll add it later because the current tip of alfonse's spec isn't fully compatible with gogl's spec parser/code generator. Currently I'm working on it.

**Comment by metaleap** [comment]
I'm curious in this matter, I specifically want to bump to GL 4.3 (now that my GeForce driver supports it!) can I download alfonse's specs and run the gogl command myself to generate a 4.3 binding or is there still an incompatibility between "the current tip of alfonse's spec" and gogl's spec parser / code generator?

**Comment by chsc** [comment]
The parser still has some problems with the new spec format (lots of warnings while parsing the spec). But theoretically it should generate working 4.3 packages. Please try it. If it works I'll add them to gogl.

**Comment by metaleap** [comment]
Seems like you're using the spec files instead of the xml files now... dang, downloading the zip with xml files in it was way more convenient... OK will try the "make" concoction then  ;)

**Comment by metaleap** [comment]
Gives me this:

```
C:\gd\src\github.com\go3d\go-gl>make download_alfonse
make: *** No rule to make target `download_alfonse'.  Stop.

C:\gd\src\github.com\go3d\go-gl>make gen_alfonse
make: *** No rule to make target `gen_alfonse'.  Stop.
```

Hmm... what can I try next?

**Comment by chsc** [comment]
Have you build gogl correctly?

These are the typical steps to build and install the bindings:

```
go build
make download_alfonse
make gen_alfonse
make install_all
```

**Comment by metaleap** [comment]
Stupid user error on my part. Didn't call make from inside the gogl dir containing the make files. Facepalm...

**Comment by metaleap** [comment]
OK **gogl -dir=alfonse_specs** re-generates all package sub-dirs but does not create a gl43 package as of yet...

**Comment by chsc** [comment]
Have you downloaded the latest spec file? You can check this by searching for a VERSION_4_3 category in gl.spec.

You can download the latest tip with: 

```
gogl -download -url=https://bitbucket.org/alfonse/gl-xml-specs/raw/tip/glspecs -dir=alfonse_specs
```

**Comment by metaleap** [comment]
Well it does have "newcategory: VERSION_4_3" at line 8412.

But OK will try the tip instead of the raw/963cd14e86d11e87be7cc976faee511fd008c038 one from the Makefile...

**Comment by metaleap** [comment]
OK with the tip one it worked. Now I have two packages, gl43 and gl43c, seemingly identical. What's the difference, which one should one use?  :)

**Comment by chsc** [comment]
The gl43c package contains old deprecated functions, but the current package grouping algorithm dosn't work correctly. Fix is coming soon.
I'll close this pull request because I have these changes already in my current development tip.

**Pull request closed without merging**